external-crates/move/crates/move-vm-runtime/src/tracing2/tracer.rs

```diff
@@ -0,0 +1,1792 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(not(feature = "gas-profiler"))]
```

Comment:
Would it make sense to rename this feature?

Reply:
Yes absolutely! But I propose this be done in a future PR, since this will be taking over the gas profiler functionality (and this is a publicly visible/usable feature flag that people use when installing the Sui CLI) so we'll need to be careful on how we rename it.